torch/_inductor/kernel/mm_scaled_grouped.py

```diff
@@ -441,38 +532,58 @@ def tuned_scaled_grouped_mm(
 
     _, is_nonzero = _is_static_problem(layout)
 
-    if is_nonzero and can_use_triton_kernel(mat_a, mat_b, offs, bias):
-        m, k1 = m1_size
-        g, k2, n = m2_size
-        k = V.graph.sizevars.guard_equals(k1, k2)
+    if is_nonzero and can_use_triton_kernel(mat_a, mat_b, offs, bias, scale_result):
```

Comment:
where are you checking that a and b are row- and column- major respectively? Do you rely on meta function to check that?

Reply:
Yes, but - do you think guards should be put here too to check on strides for row-major/column-major ordering?

Reply:
It's fine to not double check on those, but can you add a comment saying that you are relying on meta function checks so that it's easier to keep in sync?

Reply:
Done.